src/Sarif.FunctionalTests/UpdateBaselines.ps1

```diff
@@ -49,12 +49,12 @@ function Build-Baselines($toolName)
         # Actually run the converter
         Remove-Item $outputTemp -ErrorAction SilentlyContinue
         & Write-Host "$utility convert ""$input"" --tool $toolName --output ""$outputTemp"" --pretty-print --insert TextFiles"
-        &$utility convert "$input" --tool $toolName --output "$outputTemp" --pretty-print --insert TextFiles
+        &$utility convert "$input" --tool $toolName --output "$outputTemp" --pretty-print
 
         # Next, perform some rewriting. The PREfast converter in particular cannot embed file contents as the source      
         # SARIF emitted by the compiler does not contain the optional 'files' member of the 'run' object.
-        Write-Host "$utility rewrite ""$outputTemp"" --output ""$outputTemp"" --pretty-print --insert ""TextFiles;Hashes"" --force"
-        &$utility rewrite ""$outputTemp"" --output ""$outputTemp"" --pretty-print --insert "TextFiles;Hashes" --force
+        Write-Host "$utility rewrite ""$outputTemp"" --output ""$outputTemp"" --pretty-print --insert ""TextFiles;Hashes;CodeSnipets"" --force"
```

Comment:
`CodeSnipets` => `CodeSnippets` (and this is why I've never liked these echoes, here and on Line 51. I'd prefer to just remove them).

Reply:
But anyway, lovely proof of the usefulness of your "`--insert`" design.

Reply:
zapped them

---
In reply to: [202090806](https://github.com/Microsoft/sarif-sdk/pull/942#discussion_r202090806) [](ancestors = 202090806)